src/server/game/DungeonFinding/LFGMgr.cpp

```diff
@@ -2104,15 +2104,25 @@ namespace lfg
         LfgTeleportError error = LFG_TELEPORTERROR_OK;
 
         if (!player->IsAlive())
+        {
             error = LFG_TELEPORTERROR_PLAYER_DEAD;
+        }
         else if (player->IsFalling() || player->HasUnitState(UNIT_STATE_JUMPING))
+        {
             error = LFG_TELEPORTERROR_FALLING;
+        }
         else if (player->IsMirrorTimerActive(FATIGUE_TIMER))
+        {
             error = LFG_TELEPORTERROR_FATIGUE;
+        }
         else if (player->GetVehicle())
+        {
             error = LFG_TELEPORTERROR_IN_VEHICLE;
-        else if (player->GetCharmGUID())
-            error = LFG_TELEPORTERROR_CHARMING;
+        }
+        else if (player->GetCharmGUID() || player->IsInCombat())
```

Comment:
I don't know if you can get charmed without being put in combat, but if that's the case then the message thrown has always been wrong. 

Keeping as is just incase, rather than only using  `player->IsInCombat()`